pkg/controller/servicebindingrequest/reconciler.go

```diff
@@ -141,6 +142,10 @@ func (r *Reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 		logger.Info("Searching DeploymentConfig objects matching labels")
 
 		deploymentConfigListObj := &osappsv1.DeploymentConfigList{}
+		err = utils.WaitUntilResourcesFound(r.client, &searchByLabelsOpts, deploymentConfigListObj)
```

Comment:
shouldn't this be added to `default` case as well in addition to `deploymentconfig` switch case?

Reply:
@DhritiShikhar yes need to add wait to the `default` case

Reply:
Done https://github.com/redhat-developer/service-binding-operator/pull/101/commits/78eb64a5588760d876c29374da182335e4a030b0